src/content/pages/visa.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Visa Information
+subtitle: If you want to attend EuroPython 2025 in Prague, the Czech Republic, you should first verify whether you need a visa to enter the Czech Republic, and apply for one as soon as possible if you do.
+---
+
+# Visa Information
+
+## Requirements
+
+If you want to attend EuroPython 2025 in Prague, the Czech Republic, you should first verify whether you need a visa to enter the Czech Republic, and apply for one **as soon as possible** if you do.  It's also advisable to **secure a visa appointment well in advance of your planned application date** due to high demand and potential delays some people encountered last year.
+
+[This list of countries](https://www.mzv.cz/jnp/en/information_for_aliens/short_stay_visa/list_of_states_whose_citizens_are/index.html) defines if you require a visa to visit the Czech Republic.
+
+**NOTE**: The Czech Republic is part of the EU, and part of the Schengen Area. So if you already have a valid Schengen visa, you may **NOT** need to apply for an Czech visa. Please check the website above and consult your local consular office or embassy, if you are uncertain.
+
+If you do need a visa to attend EuroPython 2025, you can lodge a visa application issued for **[Short Stay (C), up to 90 days, for the purpose of “Business /Conference"](https://www.mzv.cz/jnp/en/information_for_aliens/short_stay_visa/conference.html)**. We recommend you do this as soon as possible.
+
+Please make sure you read all the visa pages carefully and prepare all the required documents before making your application. **The EuroPython organisers are not able nor qualified to give visa advice.**
+
+Should you require a visa to travel to the Czech Republic, even though it is still possible to submit your visa application 15 days before your planned travel date, it is strongly advisable to apply **as soon as possible**, as it can sometimes take more than a month to receive a decision.
+
+## Visa Support Letter
+
+Every registered attendee is welcome to request a visa support letter issued by the EuroPython Society, should you need one for your visa application!
+
+Simply fill in the form **1 week before your planned visa application**:
+
+<div class="text-center">
+<ButtonLink url="https://forms.gle/fJPPgcKFccFmAe2g9"> Visa Support Letter Request Form</ButtonLink>
+</div>
+
+We will send you the completed letter via email.
+
+Please note that we only issue visa support letters to confirmed attendees. We kindly ask you to purchase your ticket before filling in the request form. If your company purchased the ticket on your behalf, please ask them to register you as an attendee, so you have the order code ready for the form.
+
+If you find your visa application unsuccessful post the deadline for a refund (14th June), you can submit a special refund request by writing to [refunds@europython.eu](mailto:refunds@europython.eu).
```

Comment:
The deadline for refund is the 13th of June, no? It is a bit ambiguous here because 14th of June is the day post the deadline :D

Reply:
All of these should be resolved now 🙂

Reply:
Also updated the form for 3